src/crinder.cr

```diff
@@ -106,18 +106,28 @@ class Crinder::Base(T)
   end
 
   macro __process
-    def self.render(object : T) : String
-      @@object = object
+    def self.render(object : Array(T) | T) : String
```

Comment:
How about split this into two methods?
`self.render(array : Array(T))` and `self.render(object : T)`